app/views/mailers/room_ticket_mailer/send_tdx_ticket.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    Message: <b><%= raw(@message) %></b>
```

Comment:
Just a suggestion. Put information about the building, room and submitter above the message. What do you think?